### PR TITLE
New Col for Upperclassmen 

### PR DESCRIPTION
**Pull request by saucetray**
Ex: 30/44 will be 30 upperclassmen signed out of 44.

### CODE REVIEW

File: packet/models.py [hunk 1]
```diff
@@ -57,6 +57,19 @@ def signatures_required(self):
...
+        eboard = UpperSignature.query.filter_by(eboard=True).count()
+
+        return {'eboard': eboard, 'upperclassmen': len(self.upper_signatures) - eboard}
+
+
+    def upperclassmen_recieved(self):
+        upper_count = UpperSignature.query.with_parent(self).filter_by(signed=True, eboard=False).count()
+        eboard_count = UpperSignature.query.with_parent(self).filter_by(signed=True, eboard=True).count()
+
+        return {'eboard': upper_count, 'upper_count': eboard_count}
```
Comment (devinmatte) [inline packet/models.py]: You've swapped the returns

File: packet/packet.py [hunk 2]
```diff
@@ -67,6 +67,16 @@ def get_number_required(freshman_username):
     return Freshman.query.filter_by(rit_username=freshman_username).first().current_packet().signatures_required()
 
 
+@lru_cache(maxsize=2048)
+def get_number_upper_signed(freshman_username):
+    return Freshman.query.filter_by(rit_username=freshman_username).first().current_packet().upperclassmen_recieved()
+
+
+@lru_cache(maxsize=4096)
```
Comment (devinmatte) [inline packet/packet.py]: Please lower this to `2048`. I want to avoid anything above that going forward

File: packet/routes/shared.py [hunk 3]
```diff
@@ -45,7 +47,7 @@ def packets(info=None):
             packet.did_sign = False
             packet.total_signatures = sum(packet.signatures_received().values())
             packet.required_signatures = sum(packet.signatures_required().values())
-
+
```
Comment (devinmatte) [inline packet/routes/shared.py]: Remove this spacing change

File: packet/templates/active_packets.html [hunk 4]
```diff
@@ -42,6 +43,13 @@ <h3 class="page-title">Active Packets</h3>
                                                             {{ packet.total_signatures }}/{{ packet.required_signatures }}
                                                         {% endif %}
                                                     </td>
+                                                    <td>
```
Comment (devinmatte) [inline packet/templates/active_packets.html]: Add `data-sort` like on line 39 for sorting

File: packet/templates/packet.html [hunk 5]
```diff
@@ -26,6 +26,9 @@ <h3>{{ freshman.name }}</h3>
                 <div class="col">
                     <h5>Signatures: {{ signed }}/{{ required }}</h5>
                 </div>
+                <div class="col">
```
Comment (devinmatte) [inline packet/templates/packet.html]: Can you please add a screenshot showing how this is displayed?

Comment (saucetray) [reply packet/templates/packet.html]: Can't do that.

**Pull request closed without merging**